content/influxdb/v2.2/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,103 @@ menu:
+
+#### Technical preview: Durable subscription to replicate data
+
+- Add the option to [Replicate data from InfluxDB OSS to InfluxDB Cloud](/influxdb/v2.2/write-data/replication).
+
+  This [**technical preview**](/influxdb/v2.1/reference/glossary/#technical-preview) feature allows you to create a durable subscription on a per bucket basis for the purposes of replicating data from an InfluxDB OSS instance to InfluxDB Cloud. 
+  -  Allow users to store, analyze, and aggregate data locally while also forwarding newly arriving data to a centralized InfluxDB Cloud account. 
+  -  If InfluxDB OSS loses connectivity to InfluxDB Cloud, the data gets sent when network connectivity is reestablished.
+  -  Configuration options include a maximum buffer size and data age restrictions to restrict the amount of data stored on disk.
+
+#### Flux updates
+
+- Update to [Flux v0.150.0](/flux/v0.x/release-notes/#v01500-2022-01-19).
+- Add option to log Flux queries that are cancelled because of server shutdown.
+
+#### Build maintenance
+
+- Upgrade `protobuf` library.
+
+#### Task metadata
+
+- Add option to pass `type=basic` to [`/api/v2/tasks`](/influxdb/v2.1/api/#tag/Tasks) to return task metadata without the query text.
+
+#### Metrics
+
+- Add Prometheus metrics for TSM compactions, including:
+   - `storage_compaction`
+   - `storage_compactions_active`
+   - `storage_compactions_duration_seconds_bucket`
+   - `storage_compactions_duration_seconds_count`
+   - `storage_compactions_duration_seconds_sum`
+   - `storage_compactions_queued`
+   - `storage_compactions_total`
+
+- Add the following metrics tracking when data fails to be added to the replication queue:
+   - `PointsFailedToQueue`
+   - `BytesFailedToQueue`
+
+- Add the following metrics for writes, to indicate the status:
+  - `pointsWriteRequested`
+	- `pointsWriteOk`
+	- `pointsWriteDropped`
+	- `pointsWriteErr`
+	- `timeout`
+
+- Add disk size metrics per shard:
+  - `total`: number of files per shard
+  - `disk_bytes`: data size in bytes for each shard
+
+- Add cache subsystem metrics:
+  - `inuse_bytes`: Current memory consumption of cache
+  - `diskBytes`: Size of most recent snapshot"
+  - `latest_snapshot`: Unix time of most recent snapshot
+  - `writes_total`: Counter of all writes to cache
+  - `writes_err`: Counter of failed writes to cache
+  - `writes_dropped`: Counter of writes (with any dropped points) to cache
+
+- Add WAL subsystem metrics:
+  - `size`: Size of WAL in bytes
+  - `writes`: Number of write attempts to the WAL
+  - `writes_err`: Number of failed write attempts to the WAL
+
+See [InfluxDB OSS metrics](/influxdb/v2.1/reference/internals/metrics/) for additional detail.
+
+#### Display runtime configuration in use by `influxd`
+
+- Update `api/v2/config` endpoint to display the correct runtime configuration (for example, when you run `influxd print-config`).
+
+### Bug fixes
```

Comment:
Think we need this added: https://github.com/influxdata/influxdb/pull/23219
@kelseiv how are these usually populated? Is it manual?

Reply:
yes, @samhld, manual, unfortunately. We've had dreams of automating some of this process, some day.